src/classes/VOL_StateCountryPicklists.cls

```diff
@@ -80,7 +80,9 @@ public with sharing class VOL_StateCountryPicklists {
 
 			// try to deal gracefully with State            
         	string strState = (string)sobjSrc.get(strFieldPrefixSrc + 'State');        	
-            if (strState != null) {             
+            if (strState != null) {
+                // best to make sure first letter is capitalized.
+                strState = strState.capitalize();
```

Comment:
no, because we use some of the maps to refill fields.  and we don't want to convert New York into NEW  YORK.